library/cmac.c

```diff
@@ -503,107 +503,91 @@ static const  unsigned int  aes_message_lengths[NB_CMAC_TESTS_PER_KEY] = {
 };
 
 /* AES 128 CMAC Test Data */
-static const unsigned char aes_128_key[16] = {
-    0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6,
-    0xab, 0xf7, 0x15, 0x88, 0x09, 0xcf, 0x4f, 0x3c
+static const uint32_t aes_128_key[4] = {
```

Comment:
With this change the library does not seem to build correctly. The problem is that the line:
```C
#include <stdint.h>
```
is missing.